.github/workflows/super-linter.yml

```
@@ -11,10 +11,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Lint code base
-        uses: docker://github/super-linter:v2.2.2
+        uses: docker://github/super-linter:v4
```

```suggestion
        uses: github/super-linter@v4
```

---

Even better, since we don't use rust, dotenv, armttk, **pwsh**, or c# linters, we could use:
```suggestion
        uses: github/super-linter/slim@v4
```
I did a quick advanced search on GitHub for powershell files, correct me if we have some.

---

slim@v4 (= super-linter light) seems like good or even better alternative.